grammar.js

```diff
@@ -484,12 +484,14 @@ module.exports = grammar({
     catch_clause: $ => seq(
       keyword('catch'),
       '(',
-      field('type', $._type_name),
+      field('type', choice($._type_name, $.type_list)),
```

Comment:
I think it would be better to always parse this as a `type_list`, instead of having a different tree structure when there is exactly 1 type. What do you think of that?

Reply:
That's a good point. I'll change it in a day or two